### PR TITLE
use a new web font to change the displayed icon in the map markers

### LINKED ISSUE #157: Use logo for map markers
The [basket logo](https://github.com/wo-ist-markt/wo-ist-markt-media/tree/master/assets) should be used for the map markers instead of the _shopping cart_ icon.

Basket

wim-map-markers

### PR DESCRIPTION
**Pull request by tursics**
resolves #157

### CODE REVIEW

File: css/wo-ist-markt.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+       url('../font/wo-ist-markt.woff?25033009') format('woff'),
+       url('../font/wo-ist-markt.ttf?25033009') format('truetype'),
+       url('../font/wo-ist-markt.svg?25033009#wo-ist-markt') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+/* Chrome hack: SVG is rendered more smooth in Windozze. 100% magic, uncomment if you need it. */
+/* Note, that will break hinting! In other OS-es font will be not as sharp as it could be */
+/*
+@media screen and (-webkit-min-device-pixel-ratio:0) {
```
Comment (johnjohndoe) [inline css/wo-ist-markt.css]: Please remove dead code.

File: css/wo-ist-markt.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  font-style: normal;
+  font-weight: normal;
+  speak: none;
+ 
+  display: inline-block;
+  text-decoration: inherit;
+  width: 1em;
+  margin-right: .2em;
+  text-align: center;
+  /* opacity: .8; */
```
Comment (johnjohndoe) [inline css/wo-ist-markt.css]: Please remove dead code here and below.

File: js/main.js [hunk 3]
```diff
@@ -28,11 +28,11 @@ var DAY_INDEX = (now.getDay() + 6) % 7;  // In our data, first day is Monday
 var DAY_NAMES = ['Sonntag', 'Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag'];
 var DEFAULT_MARKET_TITLE = 'Markt';
 
-L.AwesomeMarkers.Icon.prototype.options.prefix = 'fa';
-var nowIcon = L.AwesomeMarkers.icon({markerColor: 'green', icon: 'shopping-cart'});
```
Comment (johnjohndoe) [inline js/main.js]: Do we need to remove the old `shopping-cart` icon somewhere?

File: css/wo-ist-markt.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
+@font-face {
+  font-family: 'wo-ist-markt';
+  src: url('../font/wo-ist-markt.eot?25033009');
+  src: url('../font/wo-ist-markt.eot?25033009#iefix') format('embedded-opentype'),
+       url('../font/wo-ist-markt.woff2?25033009') format('woff2'),
+       url('../font/wo-ist-markt.woff?25033009') format('woff'),
```
Comment (johnjohndoe) [inline css/wo-ist-markt.css]: Can you document in your commit message details what the individual file format are for? You can update your commit like this:

``` bash
git commit --amend
// update commit message
git push --force origin basket-icon
```

What is the `?25033009` for?

File: js/main.js (hunk 3 above)
Comment (tursics) [inline js/main.js]: nope

File: css/wo-ist-markt.css (hunk 4 above)
Comment (johnjohndoe) [reply css/wo-ist-markt.css]: Thanks. Found it in the commit message.

**Comment by torfsen** [comment]
Awesome! @tursics: Could you provide a quick outline of how the web font was generated? In case somebody else needs to maintain it in the future.

**Comment by johnjohndoe** [comment]
@torfsen You can find the instruction in the wo-ist-markt-media repository.

**Pull request merged**